static/app/views/detectors/components/details/common/ongoingIssues.tsx

```diff
@@ -1,82 +1,346 @@
+  );
+
+  if (isPending) {
+    return <LoadingIndicator />;
+  }
+
+  if (!group || isError) {
+    return (
+      <SimpleTable>
+        <SimpleTable.Empty>
+          <EmptyStateWarning small>
+            {t('Failed to load the latest issue')}
+          </EmptyStateWarning>
+        </SimpleTable.Empty>
+      </SimpleTable>
+    );
+  }
+
+  return (
+    <StyledTable>
+      <SimpleTable.Header>
+        <SimpleTable.HeaderCell>{t('Issue')}</SimpleTable.HeaderCell>
+        <SimpleTable.HeaderCell>{t('Status')}</SimpleTable.HeaderCell>
+        <SimpleTable.HeaderCell>{t('Last Seen')}</SimpleTable.HeaderCell>
+        <SimpleTable.HeaderCell>{t('Assignee')}</SimpleTable.HeaderCell>
+      </SimpleTable.Header>
+
+      <SimpleTable.Row>
+        <EventOrGroupCell>
+          <EventOrGroupHeader data={group} />
+        </EventOrGroupCell>
+        <SimpleTable.RowCell>
+          <GroupStatusTag fontSize="md">{group.substatus ?? group.status}</GroupStatusTag>
+        </SimpleTable.RowCell>
+        <SimpleTable.RowCell>
+          <TimeAgoCell date={group.lastSeen} />
+        </SimpleTable.RowCell>
+        <SimpleTable.RowCell>
+          <IssueAssigneeSelector group={group} />
+        </SimpleTable.RowCell>
+      </SimpleTable.Row>
+
+      <SimpleTable.Row>
+        <div style={{gridColumn: '1 / -1', paddingTop: 0}}>
+          <OpenPeriodsSubTable groupId={group.id} onZoom={zoomToRange} />
+        </div>
+      </SimpleTable.Row>
+    </StyledTable>
+  );
+}
+
+function IssueAssigneeSelector({group}: {group: Group}) {
   const organization = useOrganization();
+  const {handleAssigneeChange, assigneeLoading} = useHandleAssigneeChange({
+    organization,
+    group,
+  });
 
-  const {selection} = usePageFilters();
-  const {start, end, period} = selection.datetime;
-  const timeProps =
-    start && end
-      ? {
-          start: getUtcDateString(start),
-          end: getUtcDateString(end),
-        }
-      : {
-          statsPeriod: period,
-        };
-
-  const queryParams = {
-    ...(query || timeProps),
-    query: `is:unresolved detector:${detectorId}`,
-    limit: 5,
-  };
+  return (
+    <AssigneeSelector
+      group={group}
+      assigneeLoading={assigneeLoading}
+      handleAssigneeChange={handleAssigneeChange}
+      showLabel
+    />
+  );
+}
 
-  const issueSearch = {
-    pathname: `/organizations/${organization.slug}/issues/`,
-    query: queryParams,
+interface OngoingIssueProps {
+  detector: Detector;
+  /**
+   * Helps the zoom function add padding on left and right of the open period.
+   * If intervalSeconds is 24 hours, we would want a lot more padding than if it's 1 minute.
+   */
+  intervalSeconds?: number;
+}
+
+export function DetectorDetailsOngoingIssues({
+  detector,
+  intervalSeconds,
+}: OngoingIssueProps) {
+  const organization = useOrganization();
+  const location = useLocation();
+  // TODO: We'll probably need to make a query to get all linked issues
```

Comment:
Yeah this will be necessary for crons which has an issue for each environment


Errors we may have to do a custom `is:unresolved issue.type:error` because we don't have DetectorGroup linkages